content/docs/rfcs/20/README.md

```diff
@@ -0,0 +1,113 @@
+---
+slug: 20
```

Comment:
Add to menu as well so it shows up in preview https://github.com/vacp2p/rfc/blob/master/content/menu/index.md